### PR TITLE
Add no-relative-imports eslint rule

### PR DESCRIPTION
**Pull request by mythmon**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -70,6 +70,7 @@
     "eslint-plugin-no-unsanitized": "3.1.4",
     "eslint-plugin-prettier": "3.1.4",
     "eslint-plugin-react": "7.21.5",
+    "eslint-plugin-rulesdir": "^0.1.0",
```
Comment (rehandalal) [inline package.json]: pin this?

File: eslint-rules/no-relative-imports.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                    path.join(path.dirname(filename), importSpecifier),
+                  );
+                  const rootedImport = path.join(
+                    root,
+                    path.relative(rootPath, absoluteFsImport),
+                  );
+                  return fixer.replaceTextRange(
+                    [node.source.start + 1, node.source.end - 1],
+                    rootedImport,
+                  );
```
Comment (rehandalal) [inline eslint-rules/no-relative-imports.js:43]: 👏🏽

**Review by rehandalal — APPROVED** [review]
this is neat! nice work!!

r+wc

**Comment by rehandalal** [comment]
bors d=@mythmon

File: package.json (hunk 1 above)
Comment (mythmon) [reply package.json]: *sigh*, now I need to add a lint check for this. Or I wonder if we can configure yarn to just do this automatically...

**Comment by mythmon** [comment]
bor r= @rehandalal

File: .yarnrc [hunk 3]
```diff
@@ -0,0 +1 @@
+save-prefix ""
```
Comment (rehandalal) [inline .yarnrc:1]: 😱 this is a game changer!

Comment (mythmon) [reply .yarnrc:1]: You can even set it globally if you do it in `~/.yarnrc`!

**Review by rehandalal — APPROVED** [review]
bors r+

**Comment by mythmon** [comment]
bors r= @rehandalal

**Pull request merged**